quarkchain/cluster/client.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import print_function
```

Comment:
didn't see this being used anywhere. can remove?

Reply:
Done.